presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -1404,6 +1414,34 @@ void PrestoServer::reportNodeStatus(proxygen::ResponseHandler* downstream) {
   http::sendOkResponse(downstream, json(fetchNodeStatus()));
 }
 
+void PrestoServer::handleGracefulShutdown(const std::vector<std::unique_ptr<folly::IOBuf>>& body , proxygen::ResponseHandler* downstream){
+    std::string requestBody;
+    for (const auto& buf : body) {
```

Comment:
Isn't it enough to read the first element of the body vector and check that the size of the vector == 1?

Reply:
Done.